agent-config.json

```diff
@@ -5,7 +5,7 @@
     "minElements": 40
   },
   "anomalous-value" : {
-    "standardDevation" : 3
+    "standardDeviation" : 3
```

Comment:
Should the `anomalous-value` handler be modified to use this value?
or
Should this configuration value be deleted altogether?

Reply:
There's a follow-on ticket queued for the anomalous-value handler to use this value